persistence/docs/CHANGELOG.md

```diff
@@ -7,31 +7,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.0.0.0] - 2022-08-08
+## [0.0.0.0] - 2022-09-22
```

Comment:
If you're going to move / consolidate `changelog.md` to `persistence/docs/` then `README.md` should be moved as well